Makefile

```diff
@@ -50,15 +44,6 @@ isort: venv
 format: venv
 	@$(TOX) -e isort-fix,black-fix
 
-docs: venv
-	@$(TOX) -e docs
-
-docker:
-	$(DOCKER_COMPOSE) run --rm app bash
-
-docker/%:
-	$(DOCKER_COMPOSE) run --rm app make $*
```

Comment:
Just sharing that I actually liked the idea being able to tests from docker using the `Makefile`. Maybe docker-compose was too much, but docker I definitely like offering the option.

Reply:
I also liked it, but it is hard to do with packages supporting multiple versions of python: base python docker images contain only one version of python while running `make docker/test` leads to calling `tox -e py36,py37` in this case.

current base image is based on `ubuntu:17.04` and abuses the fact that there were 3 different versions of python available at the same time in default repo(`py27`, `py35` and `py36`) and we are using this package mainly with `py37` aiming to support latest versions of python as well. so, basically, current implementation of `docker` make target does not work and is not being used(we are using `tox-travis` plugin for tests instead of make).

Reply:
Thanks fore the feedback. Yes I think it's fine to take it out or addressed in another PR if we find a more satisfying approach